src/styles/components/_Buttons.scss

```diff
@@ -0,0 +1,116 @@
+.Btn {
+	@include font-heading;
+	@include text-std;
+	border: 1px solid $hm-primary;
```

Comment:
I think these can just be removed. I usually do this to try and make sure they are the same height as the form input fields, but this isn't required as we're using `* { box-sizing: border-box }`